updated logo of vocabulary

Dishebh opened the pull request:
## Fixes
<!-- If PR doesn't fully resolve the issue, replace 'Fixes' below with 'Related to'. -->
<!-- If there is no issue being resolved, open one before creating this pull request. -->
Fixes #220  by @Dishebh

## Description
<!-- Concisely describe what the pull request does. -->
Updated logo of vocabulary

## Screenshots
<!-- Add screenshots to show the problem and the solution; or delete the section entirely. -->
![Screenshot from 2020-05-28 21-56-04](https://user-images.githubusercontent.com/43489708/83167744-1f110c80-a12e-11ea-8ac1-734849daa8db.png)


## Checklist
<!-- Replace the [ ] with [x] to check the boxes. --> 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin
<!-- You must read and understand the following attestation. -->

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

Dishebh committed:
updated logo of vocabulary

Dishebh committed:
updated logo in readme

Dishebh commented:
@dhruvkb please review :)

dhruvkb commented on .storybook/theme.js:
```diff
@@ -27,5 +27,5 @@ export default create({
   brandTitle: 'Vue Vocabulary',
   brandUrl: 'https://opensource.creativecommons.org/cc-vue-vocabulary',
-  brandImage: 'https://raw.githubusercontent.com/creativecommons/vue-vocabulary/master/readme_assets/vocabulary_logo.svg?sanitize=true'
+  brandImage: 'logos/products/vocabulary.svg#vocabulary'
```
This won't work. It works in Vocabulary because it contains the actual SVGs, but Vue Vocabulary does not.

Dishebh replied:
I added the svg to this repo, so it worked. But I guess I didn't have to do it as we are using `vocabulary` as SSOT (as you mentioned).

dhruvkb commented on README.md:
```diff
@@ -1,17 +1,9 @@
       alt="Vocabulary logo" 
-      src="https://raw.githubusercontent.com/creativecommons/vue-vocabulary/master/readme_assets/vocabulary_logo.svg?sanitize=true"
-      height="65px"/>
+      src="https://raw.githubusercontent.com/creativecommons/vocabulary/master/readme_assets/vocabulary_logo.svg?sanitize=true"
```
```suggestion
      src="https://raw.githubusercontent.com/creativecommons/vue-vocabulary/master/readme_assets/vocabulary_logo.svg?sanitize=true"
```

dhruvkb commented on src/assets/logos/products/vocabulary.svg:
```diff
@@ -0,0 +1,18 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="212" height="40" viewBox="0 0 212 40" preserveAspectRatio="xMidYMid meet">
+  <g id="vocabulary" fill="currentColor">
+    <!-- CC -->
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M5.9329 5.79915C9.6688 2.02463 14.5032 0 19.997 0C25.4907 0 30.4352 2.02463 34.2806 5.85397C38.0715 9.62912 40.049 14.553 40.049 19.969C40.049 25.4399 38.0715 30.309 34.3356 33.9745C30.3802 37.8586 25.2705 39.9375 19.997 39.9375C14.7234 39.9375 9.72385 37.8586 5.87848 34.0293C2.03247 30.1993 0 25.2754 0 19.969C0 14.7169 2.08752 9.68395 5.9329 5 ...
+
+    <!-- Vocabulary -->
+    <path d="M59.9658 11.252H54.5938L51.6698 23.288L48.5078 11.252H42.9658L48.7458 29H54.2198L59.9658 11.252Z"/>
+    <path d="M79.1418 20.126C79.1418 17.27 78.0878 14.448 75.8778 12.646C74.2118 11.32 72.3758 10.844 70.3018 10.742C65.8138 10.47 63.3658 12.306 62.1418 13.768C60.1358 16.182 60.0678 19.072 60.0678 20.092C60.0678 23.798 61.5638 25.974 62.6178 26.994C65.0658 29.374 68.6698 29.544 69.5878 29.544C72.9198 29.544 74.9938 28.32 76.0138 27.47C77.2378 26.45 79.1418 24.138 79.1418 20.126ZM73.8378 20.296C ...
+    <path d="M97.0258 24.818L93.0818 22.778C92.8778 23.356 92.7418 23.56 92.5718 23.832C92.2658 24.274 91.3478 25.396 89.6478 25.43C88.4918 25.43 87.7438 24.852 87.4038 24.58C86.2138 23.594 85.7038 21.622 85.7038 20.092C85.7038 19.752 85.7718 16.726 87.5738 15.366C88.3558 14.788 89.1038 14.754 89.4778 14.754C91.0758 14.754 92.0618 15.842 92.3678 16.216C92.6398 16.59 92.9118 17.134 92.9458 17.236L ...
+    <path d="M114.402 29C113.96 28.354 113.892 27.776 113.892 26.246V15.876C113.892 14.856 113.722 14.38 113.518 13.904C112.328 11.354 108.962 10.776 106.48 10.776C106.072 10.776 105.222 10.776 104.202 10.912C103.59 11.014 101.652 11.286 100.19 12.714C98.8639 14.04 98.8299 15.57 98.7619 16.284H103.998C104.066 15.91 104.168 15.638 104.236 15.502C104.508 14.924 105.222 14.21 106.752 14.21C108.656 1 ...
+    <path d="M135.38 19.786C135.38 15.706 133.952 13.7 132.558 12.51C130.688 10.878 128.784 10.742 127.832 10.742C125.69 10.742 124.432 11.456 123.718 12C123.344 12.272 123.31 12.306 122.766 12.884V4.99597H117.666V29H122.528L122.494 27.13C122.8 27.504 123.106 27.844 123.514 28.15C125.112 29.408 126.948 29.476 127.662 29.476C131.742 29.476 133.68 26.552 134.292 25.294C134.768 24.342 135.38 22.438  ...
+    <path d="M153.706 29V11.252H148.504V22.302C148.504 22.778 148.47 23.22 148.3 23.662C148.062 24.308 147.178 25.26 145.512 25.26C144.73 25.26 144.288 25.022 143.982 24.852C143.03 24.274 142.996 23.424 142.996 22.302V11.252H137.726V21.826C137.726 24.002 137.556 26.858 139.528 28.422C140.922 29.51 142.894 29.51 143.302 29.51C145.478 29.51 147.042 28.626 147.688 28.116C147.892 27.946 148.096 27.74 ...
+    <path d="M162.598 29V4.99597H157.396V29H162.598Z"/>
+    <path d="M181.841 29C181.399 28.354 181.331 27.776 181.331 26.246V15.876C181.331 14.856 181.161 14.38 180.957 13.904C179.767 11.354 176.401 10.776 173.919 10.776C173.511 10.776 172.661 10.776 171.641 10.912C171.029 11.014 169.091 11.286 167.629 12.714C166.303 14.04 166.269 15.57 166.201 16.284H171.437C171.505 15.91 171.607 15.638 171.675 15.502C171.947 14.924 172.661 14.21 174.191 14.21C176.0 ...
+    <path d="M195.238 15.672V11.048C194.592 11.116 193.606 11.184 192.178 11.762C191.022 12.238 190.546 12.714 190.104 13.088V11.252H185.038V29H190.24V19.004C190.24 17.678 190.478 16.794 191.872 16.318C192.518 16.08 193.334 15.978 193.504 15.944L195.238 15.672Z"/>
+    <path d="M212 11.252H206.424L203.5 23.968L200.576 11.252H195L199.658 26.28C199.828 26.858 199.998 27.402 200.168 27.946C200.44 28.796 200.542 29.17 200.542 29.646C200.542 30.258 200.134 30.938 198.91 30.938H196.53V35.12H200.304C201.868 35.12 202.888 34.916 204.01 33.76C204.928 32.774 205.336 31.686 205.744 30.428L212 11.252Z"/>
+  </g>
+</svg>
```
We should not duplicate these assets. Makes it harder to update. We developed Vocabulary to develop a SSOT for our UI and this goes against that.

dhruvkb reviewed (commented):
Please don't delete
- `titlecard.png` until we have a replacement for it
- `package-lock.json` in any circumstances

Dishebh commented:
@dhruvkb I did some changes now:

- Removed the svg I added to the repo, and referred the `vocabulary repo` for the updated logo:
`https://raw.githubusercontent.com/creativecommons/vocabulary/master/readme_assets/vocabulary_logo.svg?sanitize=true`, so as to maintain its SSOT reputation.

- I inserted the above link in all the places wherever the older logo was used.

- I re-inserted the `titlecard.png` and `package-lock.json` files (sorry for before)

Dishebh committed:
updated vocabulary logo

dhruvkb committed:
Fix all shortcomings

dhruvkb committed:
Merge branch 'master' into logo

dhruvkb committed:
Replace class with className
